### ISSUE TITLE
WikiMiniAtlas requires title in Geohack url on Czech Wikipedia

### ISSUE THREAD

**Issue by dvorapa** [dschwen/wikiminiatlas#35]
WikiMiniAtlas refuses to load on Czech WIkipedia if there is no title parameter in Geohack url. But on enwiki it works without title

**Comment by Artoria2e5** [comment]
Yep, this line makes it clear that it won't work if params is right after `?`.
https://github.com/dschwen/wikiminiatlas/blob/ca5427505eb2c2a86936df963f02168ce18f2eb6/wikiminiatlas.js#L270

The good news is that the newer wmacore code does not seem to make this mistake any more (though it still wants to reinvent the [`searchParams` wheel](https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams)). Now how you use the new code is something I have NO IDEA about.

**Comment by dschwen** [comment]
> though it still wants to reinvent the searchParams wheel

Hahaha, that code was written almost 10 years before search params was first supported by Chrome...

**Comment by Artoria2e5** [comment]
It does seem old, and at least it works!
